app-web-frontend/src/Home/Home.js

```diff
@@ -0,0 +1,14 @@
+import Header from '../header/Header';
+
+
+function Home() {
+    return (
+        <div className="App">
+            <Header />
```

Comment:
I don't think it is a good approach to put Header under /home path. It should be accessible not only from /home path, but, for example, from /assessment for another functionality. Try to lift in on the upper level and make it render under some condition

Reply:
Goes to the backlog for next sprint